File: Recursão/C/fibonacciRecursivo.c
#include<stdio.h>

//RETORNA O N-ÉSIMO NÚMERO DA SEQUÊNCIA DE FIBONACCI

int fib(int n)
{
    if(n <= 1)
        return 1;
    return fib(n-1) + fib(n-2);
}

int main()
{
    int n = 6;
    printf("fib(%d): %d\n", n, fib(n));
}

File: Recursão/Python/fibonacciRecursivo.py
#Retorna o n-ésimo número da sequência de Fibonacci de forma recursiva

# 1 1 2 3 5 8 13 21 ...

def fib(n):
    
    return ( fib(n-1) + fib(n-2) ) if n > 1 else n


print(fib(4))